src/adagrams.js

```diff
@@ -1,15 +1,133 @@
+const generatePoolArray = (LETTER_POOL) => {
+  let poolArray = [];
+  for(let letter in LETTER_POOL){
+    for(let i = 0; i<LETTER_POOL[letter]; i++){
+      poolArray.push(letter);
+    }
+  }
+  return poolArray;
+};
+
 export const drawLetters = () => {
   // Implement this method for wave 1
+    const LETTER_POOL = {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
+    F: 2,
+    G: 3,
+    H: 2,
+    I: 9,
+    J: 1,
+    K: 1,
+    L: 4,
+    M: 2,
+    N: 6,
+    O: 8,
+    P: 2,
+    Q: 1,
+    R: 6,
+    S: 4,
+    T: 6,
+    U: 4,
+    V: 2,
+    W: 2,
+    X: 1,
+    Y: 2,
+    Z: 1,
+  };
+
+  let poolArray = generatePoolArray(LETTER_POOL);
```

Comment:
```suggestion
  const poolArray = generatePoolArray(LETTER_POOL);
```

`poolArray` isn't assigned a new value in this function so we should use `const`.